app/ui-message/client/actionButtons/messageAction.ts

```diff
@@ -1,6 +1,8 @@
 import { IUIActionButton } from '@rocket.chat/apps-engine/definition/ui';
 
+import { Utilities } from '../../../apps/lib/misc/Utilities';
 import { MessageAction, messageArgs } from '../../../ui-utils/client';
+import { t } from '../../../utils/client';
```

Comment:
It would be more "readable" if you set a alias to this import as:

`{ t as translator}`

Reply:
I agree with readability, but it is used as `t` everywhere else in the code base. So I guess consistency wins in this one? 😛

Reply:
surething